gitgud/test_commands.py

```diff
@@ -5,7 +5,31 @@
 
 
 def test_load(gg):
-    subprocess.call('git gud load basics committing', shell=True)
 
-    level = all_skills['basics']['committing']
-    assert level == gg.file_operator.get_level()
\ No newline at end of file
+    dash_temp = 'git gud load {}-{}'
+    skill_temp = 'git gud load {}'
+    nodash_temp = 'git gud load {} {}'
+    twoargs_test = [dash_temp, nodash_temp]
+    for i, skill in enumerate(all_skills):
+        for temp in twoargs_test:
+            for j, level in enumerate(skill):
+                subprocess.call(temp.format(i + 1, j + 1), shell=True)
+                assert all_skills[i][j] == gg.file_operator.get_level()
+                
+                subprocess.call(temp.format(i + 1, level.name), shell=True)
+                assert all_skills[i][j] == gg.file_operator.get_level()
+                
+                
+                subprocess.call(temp.format(skill.name, level.name), shell=True)
+                assert all_skills[i][j] == gg.file_operator.get_level()
+
+                
+                subprocess.call(temp.format(skill.name, j + 1), shell=True)
+                assert all_skills[i][j] == gg.file_operator.get_level()
+        
+        subprocess.call('git gud load {}'.format(i + 1), shell=True)
+        assert all_skills[i][0] == gg.file_operator.get_level()
+        
+        subprocess.call('git gud load {}'.format(skill.name), shell=True)
+        assert all_skills[skill.name][0] == gg.file_operator.get_level()
```

Comment:
`test_load` takes a VERY long time. You are loading/unloading every level four times. That is overkill. You just need to test each thing once, so you only need to do each of these situations only once, no iteration at all:
```
git gud load number
git gud load name
git gud load number name
git gud load name number
git gud load number-name
git gud load name-number
```
This should be the simplest thing we can test. It does one of each "half-situation", with number/name on each side of the space/hyphen.